scss/variables/typography.js

```diff
@@ -52,7 +53,113 @@ chassis.typography = {
 	sans: {
 		name: "Font Family - Sans",
 		value: "Helvetica, Arial, sans-serif"
+	},
+	link: {
+		name: "Link",
+		value: {
+			color: linkColor,
+			decoration: "none"
+		}
+	},
+	"link-hover": {
+		name: "Link On Mouse Hover",
+		value: {
+			color: linkColor,
+			decoration: "underline"
+		}
+	},
+	"link-focus": {
+		name: "Link On Focus or Active",
+		value: {
+			color: linkColor,
+			decoration: "none"
+		}
+	},
+	"link-visited": {
+		name: "Link If Visited",
+		value: {
+			color: linkColor,
+			decoration: "none"
+		}
+	},
+	hr: {
+		name: "HR style",
+		value: {
+			color: "#999",
+			thickness: "1px",
+			style: "solid"
+		}
+	},
+	code: {
+		name: "code style",
+		value: {
+			background: "#f2f2f2",
+			borderRadius: "3px"
+		}
+	},
+	pre: {
+		name: "Pre style",
+		value: {
+			background: "#f7f7f7"
+		}
+	},
+	h1: {
+		name: "H1 Style",
+		value: {
+			fontWeight: "700",
+			fontSize: "67px",
+			capitalization: "none"
+		}
+	},
+	h2: {
+		name: "H2 Style",
+		value: {
+			fontWeight: "600",
+			fontSize: "36px",
+			capitalization: "none"
+		}
+	},
+	h3: {
+		name: "H3 Style",
+		value: {
+			fontWeight: "600",
+			fontSize: "30px",
+			capitalization: "none"
+		}
+	},
+	h4: {
+		name: "H4 Style",
+		value: {
+			fontWeight: "600",
+			fontSize: "24px",
+			capitalization: "none"
+		}
+	},
+	h5: {
+		name: "H5 Style",
+		value: {
+			fontWeight: "600",
+			fontSize: "20px",
+			capitalization: "none"
+		}
+	},
+	h6: {
+		name: "H6 Style",
+		value: {
+			fontWeight: "600",
+			fontSize: "16px",
+			capitalization: "uppercase"
+		}
+	},
+	p: {
+		name: "P style",
+		value: {
+			marginTopBottom: "25px",
```

Comment:
And margin-top and margin-bottom should be different variables probably.  Same goes for marginLeftRight
Also we agreed on Tuesday to hyphenate variables and not camel case them